client.go

```diff
@@ -437,6 +445,27 @@ func (c *client) Range(ctx context.Context, r *RangeOp) ([]DomainObject, string,
 	return objectArray, token, nil
 }
 
+func (c *client) FetchRange(ctx context.Context, r *RangeOp, onNext func(value DomainObject) error) error {
```

Comment:
Well, that's not perfect, but acceptable. This forces someone to embed the function inline. There are workarounds, they can just wrap the call externally if they really need it. We should probably document this behavior.